nextcloudappstore/core/templates/user/api_token.html

```diff
@@ -0,0 +1,28 @@
+{% extends "user/base.html" %}
+{% load css_class %}
+{% load i18n %}
+{% load staticfiles %}
+
+{% block head %}
+    <script defer="defer" src="{% static 'assets/js/utils.js' %}"></script>
+    <script defer="defer" src="{% static 'assets/js/user-api-token.js' %}"></script>
+{% endblock %}
+
+{% block head-title %}{% trans "API token" %} - {% endblock %}
+
+{% block account-content %}
+    <h1>{% trans "API Token" %}</h1>
+    <section>
+        <div id="regen-success" class="alert alert-success" hidden>{% trans 'New API token generated.' %}</div>
+        <div id="regen-failure" class="alert alert-danger" hidden>{% trans 'Failed to regenerate API token.' %}</div>
+        <span id="regen-confirm-text" hidden>{% trans 'Do you want to regenerate your API token?' %}</span>
+        <p>{% trans 'Your API token is:' %}</p>
+        <p><code id="token" data-error="{% trans 'Failed to fetch token.' %}"></code></p>
```

Comment:
would prefer the same pattern as above if possible, dont parse the data-error attribute, just display the error div